prismic-model/src/parts/body.ts

```diff
@@ -138,17 +138,14 @@ export default {
           linkText: text('Button text'),
         },
       }),
-      titledTextList: slice('Titled text list', {
+      titledTextList: slice('Descriptive links list', {
         repeat: {
           title: heading('Title', { level: 3 }),
-          text: multiLineText('Text', {
-            extraTextOptions: ['heading4', 'list-item'],
-          }),
+          text: multiLineText('Text'),
```

Comment:
Could we make this a `singleLineText`?

Reply:
Good question. I'm a bit more on the fence about that one as [there are more examples where it was used](https://docs.google.com/document/d/16JzKmY7diBV-hu1bfRxIFS9WHXzUquYe1_iJDdRDoxo/edit) (over list item and headings). And after my chat with Alice around adding multi line to captioned images, I'm actually not sure why it's not done in our one instance here.

I'll ask in my appearance in the Editorial standup on Monday!

Reply:
Cool, not blocking, so happy to merge asap and revisit this later.